src/Parser/Rogue/Subtlety/Modules/Legendaries/DreadlordsDeceit.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Wrapper from 'common/Wrapper';
+
+class DreadlordsDeceit extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+	on_initialized(){
+		this.active = this.combatants.selected.hasBack(ITEMS.THE_DREADLORDS_DECEIT.id);
+  }
+  
+	item() {
+		return {
+			item: ITEMS.THE_DREADLORDS_DECEIT,
+			result: <Wrapper>Equipped.</Wrapper>,
+		};
```

Comment:
This is not enough information, please give some information about the legendary and it's effect, in this case - average stacks on cast may be interesting - could even have a suggestion tied to it perhaps?

Reply:
I think this is fine for the time being.